plugin/editorconfig.vim

```diff
@@ -260,7 +268,10 @@ function! s:EditorConfigEnable(should_enable)
     augroup editorconfig
         autocmd!
         if a:should_enable
-            autocmd BufNewFile,BufReadPost,BufFilePost * call s:UseConfigFiles()
+            autocmd BufNewFile,BufReadPost,BufFilePost * call s:UseConfigFiles(0)
```

Comment:
If the `allowEmpty` parameter of `s:UseConfigFiles()` indeed is not used, this line can stay as it was.